src/fides/api/ops/api/v1/endpoints/policy_endpoints.py

```diff
@@ -474,6 +503,11 @@ def create_or_update_rule_targets(
             detail=f"No Rule found for key {rule_key} on Policy {policy_key}.",
         )
 
+    _validate_data_categories(
+        db=db,
+        data_categories=[schema.data_category for schema in input_data],
+    )
```

Comment:
This now uses the DB session available to the API endpoint.

Reply:
It looks like this is also being checked in the `_validate_data_category` method at the model level too here? I don't know if we need both since it looks like this endpoint allows a subset to succeed and fail.

Reply:
Ah I see thanks for clarifying that you're matching existing behavior